src/main/io/cms.c

```diff
@@ -76,9 +77,9 @@
 #include "io/cms_ledstrip.h"
 
 // Forwards
-void cmsx_InfoInit(void);
-void cmsx_FeatureRead(void);
-void cmsx_FeatureWriteback(void);
+long cmsx_InfoInit(void);
```

Comment:
Not sure why you are using `long`. On ARM `long` and `int` are both 32 bit values.